plugins/woocommerce/client/admin/client/core-profiler/pages/BuilderIntro.tsx

```diff
@@ -51,41 +51,50 @@ export const BuilderIntro = ( {
 		}
 
 		apiFetch( {
-			path: '/blueprint/import',
+			path: '/wc-admin/blueprint/queue',
 			method: 'POST',
 			body: formData,
 		} )
```

Comment:
We can specify the type of response here, so we don't need to use @ts-expect-error. 🙂 

```ts
		apiFetch< {
			errors: string[];
			reference: string;
			process_nonce: string;
		} >( {
```

Reply:
Hey @chihsuan ✋ 

Thank you for the suggestions!

Changes in BuilderIntro is intended for just example. We don't have a design for this page yet. 

Could we skip reviewing this file for now?

Reply:
> We don't have a design for this page yet.

Ah sorry, we'll take blueprint out of the core profiler and the settings screen will be its main entry point. It's not quite a good fit for core profiler now that the import flow has more steps and the file can possibly take a long time to process. The settings screen gives the feature more space to expand and we can keep the core profiler streamlined.

Reply:
> Could we skip reviewing this file for now?

Sure!